CONTRIBUTING.md

```diff
@@ -40,16 +40,18 @@ Make sure all the following are true when creating a pull-request:
    ```
 
 4. Create a new branch to work on:
-   - Group your branch into a category using a prefix for your branch name, like `feature/`, `ci/`, `bugfix/`, `doc/`.
 
-      ```sh
-      git checkout -b feature/your-branch
-      ```
+    > Group your branch into a category using a prefix for your branch name, like `feature/`, `ci/`, `bugfix/`, `doc/`.
+
+   ```sh
+   git checkout -b feature/your-branch
+   ```
 
 5. Once the changes are finished, make sure there are no warnings on the code. For debugging you can [run the unit tests](#unit-tests) and [end-to-end tests](#e2e-tests).
```

Comment:
```suggestion
5. Once the changes are finished, make sure there are no warnings in the code. For debugging you can [run the unit tests](#unit-tests) and [end-to-end tests](#e2e-tests).
```

Reply:
I know not related to your changes and very nitpicky but just found this. Apart from that it lgtm.

Reply:
thanks!